pkg/server/tls_certstore_windows_test.go

```diff
@@ -154,64 +153,6 @@ func TestMatching2CertsGetMostRecent(t *testing.T) {
 	require.Equal(t, foundCert, shouldFind)
 }
 
-func TestRegularExpression(t *testing.T) {
```

Comment:
Why is this code removed? I didn't see any removal of regex logic in `tls_certore_windows.go` file.

Reply:
Before it was using it to find the subject in a complex manner, now its a more simple match.